signwriting_transcription/pose_to_signwriting/bin.py

```diff
@@ -32,8 +32,8 @@ def main():
     args = get_args()
     print('Downloading model...')
     os.makedirs("experiment", exist_ok=True)
-    hf_hub_download(repo_id=HUGGINGFACE_REPO_ID, filename=args.model, output_dir="experiment",
-                    file_name="best.ckpt")
+    hf_hub_download(repo_id=HUGGINGFACE_REPO_ID, filename=args.model, repo_type='space', local_dir='experiment')
+    os.rename(f'experiment/{args.model}', 'experiment/best.ckpt')
```

Comment:
if you must store it as `best.ckpt`, I would download the correct file, then symlink `best.ckpt` instead of renaming. That way, you can check what model file it is pointing to

Reply:
I made the changes, a general point is that I saw that collab is not support symlink, the code works but in the collab the file is best.ckpt can't open properly